ffi/diplomat/wasm/docs/decimal_ffi.rst

```diff
@@ -1,3 +1,8 @@
+..
```

Comment:
These files are autogenerated, this will fail CI.

Reply:
Should we exclude license checking for these generated documents?

Reply:
Yes